README.md

```diff
@@ -3,80 +3,101 @@
 [![Build Status](https://travis-ci.org/sourcecred/sourcecred.svg?branch=master)](https://travis-ci.org/sourcecred/sourcecred)
 [![Discord](https://img.shields.io/discord/453243919774253079.svg)](https://discord.gg/tsBTgc9)
 
-## Vision
-
-Open source software is amazing, and so are the creators and contributors who
-share it. How amazing? It's difficult to tell, since we don't have good tools
-for recognizing those people. Many amazing open-source contributors labor in
-the shadows, going unappreciated for the work they do.
-
-As the open economy develops, we need to go beyond [commit streaks] and
-follower counts. We need transparent, accurate, and fair tools for recognizing
-and rewarding open collaboration. SourceCred aims to do that.
-
-[commit streaks]: https://www.mxsasha.eu/blog/2016/04/01/how-github-contribution-graph-is-harmful/
-
-SourceCred will enable projects to create and track "cred", which is a
-quantitative measure of how much value different contributors added to a
-project. We'll do this by providing a basic data structure—a [cred graph]—into
-which projects can add all kinds of information about the contributions that
-compose it. For example, a software project might include information about
-GitHub pull requests, function declarations and implementations, design
-documents, community support, documentation, and so forth. We'll also provide
-an algorithm ([PageRank]) which will ingest all of this information and produce
-a "cred attribution", which assigns a cred value to each contribution, and thus
-to the people who authored the contributions.
-
-[cred graph]: https://en.wikipedia.org/wiki/Directed_graph
+SourceCred is an open-source system for creating reputation networks for open
+projects.  It does this by providing a Graph-based abstraction for tracking
+contributions and how contributions relate to each other, and [PageRank]
+inspired algorithms for attributing "cred" to contributions based on their
+importance.
+
+[PageRank]: https://en.wikipedia.org/wiki/PageRank
+
+#### An Example: SourceCred's cred
+
+SourceCred is initially focusing on supporting open-source projects. We think building
+tools for open-source projects to give public recognition to their contributors will
+help open-source sustainability.
+
+Naturally, we're dogfooding SourceCred on the SourceCred
+project itself. Most of our work happens on GitHub, so we import our GitHub data
+into a graph. Pulls and commits and users are all nodes, with edges connecting
```

Comment:
nit: The use of “pull” throughout is pretty hard for me to read. I’m
okay with using `PULL` instead of `PULL_REQUEST` for concision in code,
but in human-readable text this makes the result harder to understand.
Would you mind `s/pull(?! request)/pull request/g`?

Reply:
Done.